src/db.js

```diff
@@ -161,6 +161,18 @@ var Database = {
           }
         });
     }
+
+    if (process.env.NODE_ENV === 'test') {
+      // Enable mock adapter if running tests.
+      this.db.run(
+        'INSERT INTO settings (key, value) VALUES (?, ?)',
+        ['adapters.org.mozilla-iot.adapters.mock.enabled', 1],
```

Comment:
Rather than hard-coding this, I think that we should look for a particular manifest value/entry to indicate that this is a "test" plugin.